app/classes/Typolib/Typolib/Rule.php

```diff
@@ -150,6 +154,94 @@ public static function displayIfThenArrayRule()
     }
 
     /**
+     * Check a "if x then y" rule (just for ellipsis character)
+     * TODO : generic method for any character of the ifThen rule array
```

Comment:
Not sure if we should be consistent for this kind of keyword (so that we can do a quick grep to find all the occurrences), we're using FIXME in other places.